<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snowgui::exposewindow</class>
 <widget class="QWidget" name="snowgui::exposewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="snowgui::exposewidget" name="exposeWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="controlsWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="snowgui::ccdcontrollerwidget" name="ccdcontrollerWidget" native="true"/>
      </item>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="snowgui::focusercontrollerwidget" name="focusercontrollerWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>400</width>
           <height>389</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Focuser</string>
         </attribute>
        </widget>
        <widget class="snowgui::coolercontrollerwidget" name="coolercontrollerWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>400</width>
           <height>389</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Cooler</string>
         </attribute>
        </widget>
        <widget class="snowgui::filterwheelcontrollerwidget" name="filterwheelcontrollerWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>400</width>
           <height>389</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Filterwheel</string>
         </attribute>
        </widget>
        <widget class="snowgui::guideportcontrollerwidget" name="guideportcontrollerWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>400</width>
           <height>389</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Guideport</string>
         </attribute>
        </widget>
        <widget class="snowgui::adaptiveopticscontrollerwidget" name="adaptiveopticscontrollerWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>400</width>
           <height>389</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Adaptive Optics</string>
         </attribute>
        </widget>
        <widget class="snowgui::mountcontrollerwidget" name="mountcontrollerWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>400</width>
           <height>389</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Mount</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>snowgui::ccdcontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">ccdcontrollerwidget.h</header>
   <container>1</container>
   <slots>
    <signal>imageReceived()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>snowgui::guideportcontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">guideportcontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::coolercontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">coolercontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::focusercontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">focusercontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::filterwheelcontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">filterwheelcontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::adaptiveopticscontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">adaptiveopticscontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::mountcontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">mountcontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::exposewidget</class>
   <extends>QWidget</extends>
   <header>exposewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
